apollo-idling-resource/src/main/java/com/apollographql/apollo/espresso/ApolloIdlingResource.java

```diff
@@ -0,0 +1,61 @@
+package com.apollographql.apollo.espresso;
+
+
+import android.support.test.espresso.IdlingResource;
+
+import com.apollographql.apollo.ApolloClient;
+import com.apollographql.apollo.IdleCallback;
+
+import javax.annotation.Nonnull;
+
+import static com.apollographql.apollo.api.internal.Utils.checkNotNull;
+
+/**
+ * An Espresso {@link IdlingResource} for {@link com.apollographql.apollo.ApolloClient}.
+ */
+public class ApolloIdlingResource implements IdlingResource {
```

Comment:
I feel it would be better to have this be in a separate repo and linked to from apollo-android. What do you think?